cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("movies/", movie_list, name="bus_list"),
```

Comment:
The name for this path is set to 'bus_list', which seems to be a mistake. It should likely be 'movie_list' to match the view function and maintain consistency.